src/main/java/ru/lanwen/diff/uri/UriDiffer.java

```diff
@@ -28,22 +28,25 @@
  * User: lanwen
  */
 public class UriDiffer {
+
     //TODO filter
     //TODO encoding
 
-    private URI actual = URI.create("/");
-    private URI expected = URI.create("/");
+    public static final URI DEFAULT_URI = URI.create("/");
+
+    private URI actual = DEFAULT_URI;
+    private URI expected = DEFAULT_URI;
 
     public static UriDiffer diff() {
         return new UriDiffer();
     }
 
-    public UriDiffer actual(String actual) throws URISyntaxException {
-        return actual(new URI(actual));
+    public UriDiffer actual(String actual) {
+        return actual(URI.create(actual));
     }
 
-    public UriDiffer expected(String expected) throws URISyntaxException {
```

Comment:
ok for this change (i've already use `URI.create()` everywhere - think as the most of other)